src/components/navbar/Navbar.jsx

```diff
@@ -1,16 +1,17 @@
 import './Navbar.css';
 import { BrowserRouter as Router, Link, NavLink } from 'react-router-dom';
+import logo from './artparty-logo.png';
 
 const Navbar = () => {
     return (
-        <nav className="navbar">
-            <Router>
-                <Link to='/' >
-                    <img src='' alt="Art Party logo"/>
+        <Router>
+            <nav className="navbar">
+                <Link to='/' className="logo-link">
```

Comment:
I noticed that you used an `a` when you were targeting this in your `CSS` using the `a` tag I think you can still give this a class name so it is more clear when when someone looks at your `CSS`